packages/lib/src/vis/scatter/ScatterPoints.tsx

```diff
@@ -20,6 +21,7 @@ interface Props {
   colorMap: ColorMap;
   invertColorMap: boolean;
   size: number;
+  onClick?: (evt: ThreeEvent<MouseEvent>) => void;
```

Comment:
Might be worth having a `handleClick` handler in this component that calls `onClick(index, evt)`, so consumers can access the index of the point more easily.

Reply:
We could also imagine having a `handleClick` handler in `ScatterVis` to add the point's value/abscissa/ordinate. Not as crucial, though, as the consumer can easily access those with the `index`.